src/compas_fab/utilities/lazy_loader.py

```diff
@@ -0,0 +1,44 @@
+# Code copied from https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/util/lazy_loader.py
```

Comment:
This guy, brazenly copied from tensorflow, lazily loads pybullet, but it does not, at least as used, lazily load PyBulletClient when importing anything from compas_fab.backends.